crypto/s2n_hmac.c

```diff
@@ -252,14 +265,18 @@ int s2n_hmac_update(struct s2n_hmac_state *state, const void *in, uint32_t size)
      * input. On some platforms, including Intel, the operation can take a
      * smaller number of cycles if the input is "small".
      */
-    state->currently_in_hash_block += (4294949760 + size) % state->hash_block_size;
+    ENSURE_POSIX(size <= UINT32_MAX - 4294949760, S2N_ERR_INTEGER_OVERFLOW);
```

Comment:
Since 4294949760 now appears more than once, it should probably now be a named constant.

Reply:
Added as a const variable.